simulariumio/cellpack/cellpack_converter.py

```diff
@@ -32,7 +33,12 @@
 
 
 class CellpackConverter(TrajectoryConverter):
-    def __init__(self, input_data: CellpackData):
+    def __init__(
+        self,
+        input_data: CellpackData,
+        progress_callback: Callable = None,
```

Comment:
can this be typed more specifically?  Callable is totally general.  The progress callback seems to take one parameter (the percentage) and is also expected to have a call_count property?

Reply:
or is call_count a property of Mock...?

Reply:
Call count is a property of Mock, so it's just for testing. I could add a call_count parameter to the callback, but I don't think we need that. I'll certainly go ahead and be more specific for the Callable type

- [ ] make Callable type more specific (ie have 1 float param)